pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -52,30 +52,32 @@ class RandInt(Module):
 
   def create_returnn_layer_dict(self, low, high, size, dtype=None) -> Dict[str, Any]:
     dtype = dtype or "int64"
-    assert isinstance(low, int)
-    assert isinstance(high, int)
-    assert all(isinstance(sz, int) for sz in size)
+    low = self._to_int(low)
+    high = self._to_int(high)
+    size = tuple(self._to_int(sz) for sz in size)
```

Comment:
This is wrong. You do not want to cast a tensor to an `int`. You want to pass a tensor (i.e. using `self._get_input_layer_name`), or if it is an `int`, then you can pass it as it is.

Reply:
Yes, right. I think also `_get_output_shape_from_returnn` will need to be fixed, but still there is the issue in `get_module_by_abs_id_name`.